src/transforms.coffee

```diff
@@ -383,7 +383,7 @@ module.exports.makeYieldConditionally = makeYieldConditionally = (simpleLoops, w
       node.yields = true
       possiblyGeneratorifyAncestorFunction node unless node.mustBecomeGeneratorFunction
     else if node.mustBecomeGeneratorFunction
-      node.update node.source().replace /^function /, 'function* '
+      node.update "(_tmp = #{node.source().replace(/^function /, "function* ")}, _tmp._isAetherGen = true, _tmp)"
```

Comment:
Here I'm creating an implicit global `_tmp`, just to store the just-generatorified function and set a flag on it that will be used in innerator (explained in more detail at codecombat/codecombat#3433).

I'd like to know a way to do this without polluting the global scope.